scripts/bump_deps.lua

```diff
@@ -3,21 +3,38 @@
 -- Usage:
 --    ./scripts/bump_deps.lua -h
 
-local M = {}
+assert(vim.fn.executable('curl') == 1)
+assert(vim.fn.executable('sed') == 1)
+assert(vim.fn.executable('git') == 1)
 
 local _trace = false
```

Comment:
Is that still needed?

Reply:
Technically yes, practically no. We can absolutely remove `curl` since it's available everywhere. `git` should be given. I guess we can keep `sed` as it would be a surprise for windows users?

Reply:
Should I remove sed too?

Reply:
Sorry, I was talking about the `_trace` line.

The sed check can stay; it's the only command where failure would leave stale files around.

Reply:
Oh, the trace. Eh, I guess it could be useful? to see which run commands are used? Idk, Not super attached to it.

Reply:
As you can't set it from the outside, I would say "no". More code to yeet :)